### ISSUE TITLE
SyntaxWarning: invalid escape sequence

### ISSUE THREAD

**Issue by ryandesign** [doxygen/doxygen#10739]
**Describe the bug**
Building doxygen 1.10.0 with python 3.12 causes these warning to appear:

```
doxygen-1.10.0/src/configgen.py:669: SyntaxWarning: invalid escape sequence '\s'
doxygen-1.10.0/vhdlparser/vhdl_adj.py:18: SyntaxWarning: invalid escape sequence '\s'
doxygen-1.10.0/vhdlparser/vhdl_adj.py:18: SyntaxWarning: invalid escape sequence '\s'
doxygen-1.10.0/src/configgen.py:669: SyntaxWarning: invalid escape sequence '\s'
doxygen-1.10.0/src/configgen.py:669: SyntaxWarning: invalid escape sequence '\s'
build/doc/translator.py:1: SyntaxWarning: invalid escape sequence '\l'
build/doc/translator.py:1474: SyntaxWarning: invalid escape sequence '\('
doxygen-1.10.0/src/configgen.py:669: SyntaxWarning: invalid escape sequence '\s'
```

**To Reproduce**
Build doxygen 1.10.0 with python 3.12 or later

**Expected behavior**
No warnings

**Version**
doxygen 1.10.0
python 3.12.2
macOS 12.7.2

**Additional context**
See https://docs.python.org/3/library/re.html:

> The solution is to use Python’s raw string notation for regular expression patterns; backslashes are not handled in any special way in a string literal prefixed with `'r'`. So `r"\n"` is a two-character string containing `'\'` and `'n'`, while `"\n"` is a one-character string containing a newline. Usually patterns will be expressed in Python code using this raw string notation.

**Comment by ryandesign** [comment]
Excuse me, it looks like configgen.py and translator.py were already fixed in 80fab9a1ab565730a84a0e35690d0d41be1a804a and vhdl_adj.py was fixed as part of 8bc3d9dc0b3f90c881bcc2db7652c7063039a6c5.

**Comment by albert-github** [comment]
So no problem anymore?
